tests/common/mod.rs

```diff
@@ -0,0 +1,44 @@
+use bom;
+use bom::bom::options::{BitcodeOptions, ExtractOptions, Options, Subcommand};
+use std::path::{PathBuf};
+
+pub static SOURCE_DIR: &'static str = "tests/sources";
+
+
+// Get the clang command provided by the user (via the CLANG environment variable), if any
+//
+// If the user did not provide one, just return None, which is interpreted by build-bom as 'clang'
```

Comment:
If these commands are used just in the tests, does it make sense to have them return `PathBuf`/`String` and just do the default replacement here?

Reply:
Makes sense for the `user_llvm_dis_cmd`, and I changed that, but the others are actual `Option` fields in the options structures.